extlinks/programs/models.py

```diff
@@ -40,3 +40,14 @@ def any_orgs_user_list(self):
             return True
         else:
             return False
+
+    @property
+    def any_orgs_bot_edits(self):
```

Comment:
Per my comment further up, this isn't necessary for this filter. We want to know if any organisations have user lists for `any_orgs_user_list` but that's overcomplicating things for the bot edits, we'll just always show the checkbox.